src/mapblock.cpp

```diff
@@ -381,38 +383,46 @@ void MapBlock::serialize(std::ostream &os, u8 version, bool disk, int compressio
 	/*
 		Bulk node data
 	*/
+	u8 content_width = 2;
+	u8 params_width = 2;
```

Comment:
Should be `const`

Reply:
Copied from existing code... But I agree.

Reply:
done